docs/dev/design_documents/resource_before_notifications.md

```diff
@@ -13,20 +13,20 @@ Let users trigger another resource prior to an action happening.
     So that I don't unnecessarily run my preconditions on every single Chef run.
 
     As a Chef user,
-    I want Chef to run me an action if a resource updates,
+    I want Chef to run an action if a resource updates,
     So that I don't have to implement the resource test logic in my recipe.
 
 ## Specification
 
-We add a new :before timing which causes a notification to happen
+We add a new `:before` timing which causes a notification to happen
```

Comment:
I added a lot of `this type of formatting` to resource_before_notifications.md. I may have gone a bit overboard and incorrectly formatted a few things. I would appreciate being checked on this.

Reply:
those look good